lib/browser/browser.js

```diff
@@ -73,7 +73,9 @@ module.exports = class Browser {
             sessionId,
             ...sessionOpts,
             ...detectedSessionEnvFlags,
-            ...this._config.sessionEnvFlags
+            ...this._config.sessionEnvFlags,
+            options: sessionOpts,
+            capabilities: {...sessionOpts.capabilities, ...sessionCaps}
```

Comment:
It is necessary because otherwise when run tests using `devtools` protocol creates two instance of chrome browser for each test. First when call `remote` method in master process and second when call `attach` method in worker process.

The problem is here - https://github.com/webdriverio/webdriverio/blob/main/packages/devtools/src/launcher.ts#L196-L203.

When browser creates in master process it returns capabilities with - `'goog:chromeOptions': {debuggerAddress: 'localhost:12345'}` which is necessary to connect to it when call `attach` method. But capabilities from browser config don't know anything about it option. So i merge it.